src/runmodes.py

```diff
@@ -0,0 +1,5 @@
+import src.programs
+
+mode_selector = { "NNLOJET": src.programs.NNLOJET,
+                  "HEJ": src.programs.HEJ,
+}
```

Comment:
Same here, pyHepgrid.src rather than src.
Later on we can move the folders around if necessary.